pkg/build/builder/dockerutil.go

```diff
@@ -49,6 +49,26 @@ type DockerClient interface {
 	TagImage(name string, opts docker.TagImageOptions) error
 }
 
+func pullImage(client DockerClient, name string, authConfig docker.AuthConfiguration) error {
+	logProgress := func(s string) {
+		glog.V(0).Infof("%s", s)
```

Comment:
I don't know that we should be showing this - shouldn't this be consistent with @csrwng's other progress displayer for builds?